frontend/app/routes/public/renew/$id/ita/update-home-address.tsx

```diff
@@ -73,63 +73,63 @@ export async function action({ context: { appContainer, session }, params, reque
   const t = await getFixedT(request, handle.i18nNamespaces);
   const { CANADA_COUNTRY_ID, USA_COUNTRY_ID } = appContainer.get(TYPES.configs.ClientConfig);
 
-  const addressInformationSchema = z
+  const homeAddressSchema = z
     .object({
-      homeAddress: z.string().trim().max(30).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')).optional(),
-      homeCountry: z.string().trim().optional(),
-      homeProvince: z.string().trim().optional(),
-      homeCity: z.string().trim().max(100).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')).optional(),
-      homePostalCode: z.string().trim().max(100).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')).optional(),
+      address: z.string().trim().max(30).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')),
+      country: z.string().trim(),
+      province: z.string().trim().optional(),
+      city: z.string().trim().max(100).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')),
+      postalCode: z.string().trim().max(100).refine(isAllValidInputCharacters, t('renew-ita:update-address.error-message.characters-valid')).optional(),
     })
     .superRefine((val, ctx) => {
-      if (!val.homeAddress || validator.isEmpty(val.homeAddress)) {
+      if (!val.address || validator.isEmpty(val.address)) {
         ctx.addIssue({ code: z.ZodIssueCode.custom, message: t('renew-ita:update-address.error-message.home-address.address-required'), path: ['homeAddress'] });
```

Comment:
The `path` has to match the field name in the Zod schema as well as the corresponding `name` value for the input field. Otherwise, the error messages will not be rendered.

```suggestion
        ctx.addIssue({ code: z.ZodIssueCode.custom, message: t('renew-ita:update-address.error-message.home-address.address-required'), path: ['address'] });
```

(same comment for the rest of the fields below and `update-mailing-address`)

Reply:
Oh I thought those we're for the input field `name`!